src/platform/modulesmanager/src/ModulesManager.cpp

```diff
@@ -26,20 +26,16 @@
 #include <ScopeGuard.h>
 
 constexpr unsigned int g_defaultModuleCleanup = 60 * 30; // 30 minutes
+
 const std::string g_moduleDir = "/usr/lib/osconfig";
 const std::string g_moduleExtension = ".so";
 
-const char* g_configJson = "/etc/osconfig/osconfig.json";
+const std::string g_configJson = "/etc/osconfig/osconfig.json";
 const char* g_configReported = "Reported";
 const char* g_configComponentName = "ComponentName";
 const char* g_configObjectName = "ObjectName";
 
-// The local desired and reported configuration files
-const std::string g_desiredFile = "/etc/osconfig/osconfig_desired.json";
-const std::string g_reportedFile = "/etc/osconfig/osconfig_reported.json";
-
-// Manager mapping clientName <-> ModulesManager
-std::map<std::string, std::weak_ptr<ModulesManager>> manMap;
+// Temporary map used by MpiDoWork() while the Platform is still attatched to the Agent
 // Manager mapping MPI_HANDLE <-> ModulesManager

```

Comment:
Remove this comment

Reply:
The Platform is not attached to the agent. They just run in same process for now. Statement is not true

Reply:
Should I also remove the comment from `src/platform/inc/Mpi.h`?

Reply:
There is a similar comment there

Reply:
Yes, please! Thanks